packages/frontend/core/src/modules/collection/services/pinned-collection.ts

```
@@ -14,6 +14,11 @@ export class PinnedCollectionService extends Service {
     super();
   }
 
+  pinnedCollectionDataReady$ = LiveData.from(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)